html.js

```diff
@@ -28,7 +28,7 @@ class HTML extends Component {
             content="width=device-width, initial-scale=1.0"
           />
           <title>{title}</title>
-          <link rel="icon" href="favicon.ico" type="image/x-icon" />
+          <link rel="icon" href={favicon} type="image/x-icon" />
```

Comment:
That is exactly what I mean! Sorry, I made a typo yesterday
funny that it doesn't cause an error in console since you added `favicon.ico` to the project root